src/login.js

```diff
@@ -3,43 +3,91 @@ import { Checkbox } from "react-icheck";
 import "./Login.css";
 
 class Login extends  Component{
+  constructor(props){
+    super(props);
+  }
+  
+  renderUserCredentialsEnabled(){
+    if(this.props.userCredentialsEnabled){
+      return (
+        <form action="#" method="post">
+          <div className="form-group has-feedback">
+            <input type="email" className="form-control" placeholder="Email"></input>
+            <span className="glyphicon glyphicon-envelope form-control-feedback"></span>
+          </div>
+          <div className="form-group has-feedback">
+            <input type="password" className="form-control" placeholder="Password"></input>
+            <span className="glyphicon glyphicon-lock form-control-feedback"></span>
+          </div>
+          <div className="row">
+            <div className="col-xs-8">
+              <div className="remember_me">
+                <Checkbox
+                  checkboxClass="icheckbox_square-blue"
+                  increaseArea="50%"
+                  label="Remember Me"
+                />
+              </div>
+            </div>
+            <div className="col-xs-4">
+              <button type="submit" className="btn btn-primary btn-block btn-flat">Sign In</button>
+            </div>
+          </div>
+        </form>
+      );
+    } else {
+      return (<div></div>);
+    }
+  }
+  
+  renderAuthProviders(){
+    let authProviders = [];
+    if(this.props.authProviders.length > 0){
+      authProviders = this.props.authProviders.map(provider => {
+        switch (provider){
+          case "facebook":
+            return (
+              <a href="#" className="btn btn-block btn-social btn-facebook btn-flat">Name
+                <i className="fa fa-facebook"></i> Name
+                Sign in using Facebook
+              </a>
+            );
+          case "google":
+            return (
+              <a href="#" className="btn btn-block btn-social btn-google btn-flat">Name
```

Comment:
What is the `Name` text for? I don't see it in original html.

Reply:
Sorry for my editor, sometimes move that to another place (Name for className property) which I don't see while committing codes.

Reply:
Ok, still...can you please remove it?

Reply:
removed it, please pull the latest code.